src/cascadia/TerminalSettingsEditor/Launch.xaml

```diff
@@ -40,7 +40,7 @@
                 <local:SettingContainer x:Uid="Globals_DefaultProfile"
                                         Margin="0">
                     <ComboBox x:Name="DefaultProfile"
-                              ItemsSource="{x:Bind State.Settings.AllProfiles, Mode=OneWay}"
+                              ItemsSource="{x:Bind State.Settings.ActiveProfiles, Mode=OneWay}"

```

Comment:
I think this is wrong... We have to list hidden profiles right?
In that case I need to write a wrapper function real quick which filters deleted profiles.